specification/personal-demographics.yaml

```diff
@@ -998,6 +998,26 @@ paths:
         * Cannot be a date in the future.
         * Cannot be after the deceased date, if present.
 
+        ## Multiple Birth order
+
+        Single item found under `multipleBirthInteger` field.
+
+        In a JSON Patch request the path should be `/multipleBirthInteger`.
+
+        When adding or updating the birth order, the update should be an integer in the range `1` - `9` inclusive.
```

Comment:
Are we not missing something about the use of intergers 8 and 9 here @ben-clarke ?

Reply:
There is a section on the Birth Order schema (MultipleBirthOrder.yaml) that details this. However I can add it here as well.